src/test-fixtures/docmapGenerators.ts

```diff
@@ -62,6 +62,13 @@ export const fixtures = {
     return generateDocMap('test', publisher, firstStep);
   },
 
+  preprintWithPartialManuscriptWithPublishedDate: (): DocMap => {
+    const manuscript = generateManuscript('10.1101/123456', '123456', undefined, 'RP123456', undefined, new Date('2022-03-01'));
+    const preprint = generatePreprint('preprint/article1', new Date('2022-03-01'), undefined, undefined, undefined, undefined, manuscript);
```

Comment:
Just noting that the `published` field would likely be a timestamp rather than just a date.